src/lib/InfoOverlay.svelte

```diff
@@ -1,216 +1,1932 @@
 <script lang="ts">
-	import { infoOverlayVisible } from '../stores';
+	import { infoOverlayVisible, infoOverlayActiveTab } from '../stores';
 	import CloseButton from './CloseButton.svelte';
-	import ActionButton from './ActionButton.svelte';
+	import { SvelteToast, toast } from '@zerodevx/svelte-toast';
 
+	const app = new SvelteToast({
```

Comment:
I think this is not needed. Removing it doesn't seem to change anything. Then also remove the import.